features/draft_environment.feature

```diff
@@ -11,14 +11,14 @@ Feature: Draft environment
     Then I should be on the case study page
     And the page should contain the draft watermark
 
-  @draft
+  @draft @high
```

Comment:
ok thanks, do you agree with the priorities assigned to the other checks in this file? and would you mind also reviewing the ones in #150? Thanks